### PR TITLE
[DD-953] : [DD-1082]

### PR DESCRIPTION
**Pull request by boferri**
- ecord search by legacy record identifier attribute path + record id + data model URI endpoint

### CODE REVIEW

File: controller/src/main/java/org/dswarm/controller/resources/resource/DataModelsResource.java [hunk 1]
```diff
@@ -337,6 +340,123 @@ public Response updateDataModelData(@ApiParam(value = "data model identifier", r
...
+
+		if (!data.isPresent()) {
+
+			DataModelsResource.LOG.debug("couldn't find records for key attribute path '{}' and search value '{}' in data model with uuid '{}'",
+					keyAttributePathString, searchValue, uuid);
+
+			return Response.status(Status.NOT_FOUND).build();
+		}
+
+		// temp
```
Comment (knutwalker) [inline controller/src/main/java/org/dswarm/controller/resources/resource/DataModelsResource.java:426]: classic

Comment (boferri) [inline controller/src/main/java/org/dswarm/controller/resources/resource/DataModelsResource.java:426]: good? bad?

Comment (boferri) [inline controller/src/main/java/org/dswarm/controller/resources/resource/DataModelsResource.java:426]: or empty JSON array instead of 404?

Comment (knutwalker) [inline controller/src/main/java/org/dswarm/controller/resources/resource/DataModelsResource.java:426]: A reference to //temp which I expect to be everything but temporary ;-)

**Pull request merged**